safe-global/safe-wallet-web
# fix: Add AB test for human-readable transactions

## What it solves

Part of #2011 

## How this PR fixes it

- Adds A/B testing for human-readable descriptions in the history/queue
- Adds a new event for `Fetching transaction details`
- Fixes slight (4px difference) column alignment issue on the date between human-readable transactions and other ones

## How to test it

1. Open a Safe
2. Observe there is a new local storage key called `SAFE_v2__AB_human-readable`
3. Navigate to the history/queue
4. Switch the key from true to false or vice versa
5. Observe the history/queue changes layout
6. Remove the local storage key and reload the page until the value changes, if its false observe no abTest key in GTM otherwise observe the abTest key in GTM saying `human-readable`
7. Observe there is a new event `Fetch transaction details` when opening a transaction in the history for the first time

## Screenshots
<img width="1268" alt="Screenshot 2023-09-25 at 16 33 03" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/f698c445-6a8b-4af7-9739-b2e3f3fc92e7">
<img width="1271" alt="Screenshot 2023-09-25 at 16 33 18" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/fbc23c9e-94cc-4239-8123-3ae480978030">

## Checklist
